tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -0,0 +1,255 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Run kubevirt testing
+# Maintainer: Nan Zhang <nan.zhang@suse.com>
+
+use base multi_machine_job_base;
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+use Config::Simple;
+
+my @kubevirt_tests = (
+    'vmi_lifecycle_test',
+    'vm_test',
+    'vmipreset_test',
+    'vmi_configuration_test',
+    'vmi_headless_test',
+    'access_test',
+    'storage',
+    'container_disk_test',
+    'datavolume',
+    'hotplug',
+    'vmi_cloudinit_test',
+    'vmi_iothreads_test',
+    'vmi_multiqueue_test',
+    'primary_pod_network',
+    'vmi_multus',
+    'vmi_networking',
+    'networkpolicy',
+    'vmi_hostdev_test',
+    'vmi_gpu_test',
+    'vmi_monitoring_test',
+    'vnc_test',
+    'console_test',
+    'credentials_test',
+    'expose',
+    'replicaset_test',
+    'network_services',
+    'vmi_kernel_boot_test',
+    'infra_test',
+    'operator_test',
+    'migration_test',
+    'subresource_api_test',
+    'virt_control_plane_test'
+);
+
+sub run {
+    my ($self) = @_;
+    my $ip_out = script_output('ip route show | grep -Eo "src\s+([0-9.]*)\s+" | head -1 | cut -d\' \' -f 2', 30);
+    set_var('SERVER_IP', $ip_out);
+    bmwqemu::save_vars();
+
+    # Wait for destination to be ready
+    $self->set_mutex_lock('DST_READY_TO_START');
+
+    my $agent_ip = $self->get_var_from_parent("AGENT_IP");
+
+    rke2_server_setup($self, $agent_ip);
+    deploy_kubevirt_manifests();
```

Comment:
Before doing this, seems some tips given by Vasily in [second phase testing](https://confluence.suse.com/pages/viewpage.action?spaceKey=qasleapac2&title=%5BKubeVirt%5D+Second+phase+test+result+and+howto#id-[KubeVirt]Secondphasetestresultandhowto-Howtosetuptestenvironment) are missing. Please double check inline comments that I at you to keep you informed.

Reply:
Pariticularly, eg 
![Uploading Screenshot_20220301_104022.png…]()

Reply:
Besides, the local storage class is not deployed , refer to [link](https://confluence.suse.com/pages/viewpage.action?spaceKey=qasleapac2&title=%5BKubeVirt%5D+Second+phase+test+result+and+howto#id-[KubeVirt]Secondphasetestresultandhowto-RKE2incompatibleaspectswithkubevirt-tests) for details.